### ISSUE TITLE
does not work with brunch 2.7.4

### ISSUE THREAD

**Issue by mithereal** [theocodes/vue-brunch#21]
i cant get this to work on the specified version of brunch 
when  templates: {
      joinTo: "js/app.js"
    }
is commented out or the jointo is changed 8in this case to Vue.js when i uncomment/leave default i get a huge file that looks like all the js  in brunch is being build into this file, i cant use the chrome vue devtool because it says not installed also i get this error web/static/customer/components/app.vue failed. Module did not self-register. ^G
can you help 
heres my brunch config

```
exports.config = {
  // See http://brunch.io/#documentation for docs.
  files: {
    javascripts: {
        joinTo: {
        "js/front.js": /^(web\/static\/customer\/js)|(node_modules)/,
        "js/agent.js": /^(web\/static\/agent\/js)|(node_modules)/,
        "js/admin.js": /^(web\/static\/admin\/js)|(node_modules)/,
         "js/admin_vendor.js": /^(web\/static\/admin\/vendor).*/,
         "js/front_vendor.js": /^(web\/static\/customer\/vendor).*/,
         "js/agent_vendor.js": /^(web\/static\/agent\/vendor).*/,
         "js/jquery.min.js":  ["web/static/vendor/jquery.min.js"],
         "js/fullcalendar.min.js":  ["web/static/vendor/fullcalendar.min.js"],
         "js/moment.js":  ["web/static/vendor/moment.js"],
         "js/phoenix.js":  ["web/static/vendor/phoenix.js"]
        }

    },
    stylesheets: {
        joinTo: {
        "css/main.css": /^(web\/static\/css)/,
        "css/customer/app.css": /^(web\/static\/customer\/css)/,
        "css/customer/vendor.css": /^(web\/static\/customer\/vendor).*/,
        "css/agent/app.css": /^(web\/static\/agent\/css)/,
        "css/agent/vendor.css": /^(web\/static\/agent\/vendor).*/,
        "css/admin/app.css": /^(web\/static\/admin\/css)/,
        "css/admin/vendor.css": /^(web\/static\/admin\/vendor).*/,
        "css/vendor.css": /^(web\/static\/css\/vendor).*/
      },
      order: {
        after: ["web/static/css/main.css"] // concat app.css last
      }
    },
    templates: {
      joinTo: "js/App.js"
    }
  },

  conventions: {
    // This option sets where we should place non-css and non-js assets in.
    // By default, we set this to "/web/static/assets". Files in this directory
    // will be copied to `paths.public`, which is "priv/static" by default.
    assets: /^(web\/static\/assets)/
  },

  // Phoenix paths configuration
  paths: {
    // Dependencies and current project directories to watch
    watched: [
      "web/static",
      "test/static"
    ],

    // Where to compile files to
    public: "priv/static"
  },

  // Configure your plugins
  plugins: {
    babel: {
      // Do not use ES6 compiler in vendor code
        ignore: [
            /(web\/static\/vendor)|node_modules/,
            /(web\/customer\/templates\/partials)/
        ]

    },
      copycat: {
          // copies to priv/static/*
          "fonts": ["node_modules/bootstrap-sass/assets/fonts/bootstrap"],
          "fonts": ["node_modules/font-awesome/fonts"]

      },

      vue: {
          extractCSS: true,
          out: 'priv/static/css/components.css'
      }
  },



  modules: {
    autoRequire: {
      "js/front.js": ["web/static/customer/js/app"],
      "js/admin.js": ["web/static/admin/js/admin"],
      "js/agent.js": ["web/static/agent/js/agent"],
    }
  },

    npm: {
    globals: {
        $: 'jquery',
        jQuery: 'jquery',
        bootstrap: 'bootstrap',
        Vue: "vue/dist/vue.common.js"
    },
        enabled: true,
        whitelist: ["phoenix", "phoenix_html", "vue"],

        styles: {
            'normalize.css': ['node_modules/normalize.css/normalize.css']
        }
        }
};
```

**Comment by theocodes** [comment]
@mithereal could you please create a minimal bare bones project that demonstrates the issues you've had and link it here? It's always best to reduce the cognitive load when reporting issues so to reduce the guess work...

Thanks

**Comment by tomquirk** [comment]
hey @mithereal ,

The [brunch-vue-barebones](https://github.com/tomquirk/brunch-vue-barebones) skeleton currently uses brunch `2.10.x`. Perhaps check out that project to give you an idea of how to integrate vue-brunch into your own project.

Closing this for now, but feel free to create a new ticket if you think it's still an issue

**Comment by lroal** [comment]
I did check out the brunch-vue-barebones. Updated all dependencies. Using node 8.
Reload works the first time, but saving the file a second causes a crash in the browser.

**Comment by tomquirk** [comment]
If I had to guess, I'd say something in your brunch-config is causing it to break. Unfortunately, there's not much I can do without more info! See if you can narrow down the problem and create a new Issue 😄

**Comment by lroal** [comment]
I am just using the skeleton as it is. It fails to reload if you save a file a second time:
_brunch new -s vue_
update all depencies to *   
_npm install_
_npm start_ 
open localhost at port 3333
change the msg in Hello.vue and save it  
reload the web page (it works fine)  
change the msg in Hello.vue again and save it  
reload the web page (it crashes)   .

**Comment by lroal** [comment]
It might not be related to @mithereal 's problem, but it is still very frustrating when not even the skeleton project works. That makes me want to give up brunch and go for good old webpack and browserify.

**Comment by tomquirk** [comment]
Hmm okay, definitely sounds like there's a bug somewhere. Is there anything to suggest it is in this project and not the skeleton? 

Either way, @lroal, your issue doesn't sound like its the same as this issue, so I'd suggest making another Issue in [brunch-vue-barebones](https://github.com/tomquirk/brunch-vue-barebones). I maintain it so if it turns out the issue is in vue-brunch I'll recreate it here.

**Comment by lroal** [comment]
Okay, I done that now.
The skeleton [tim15/brunch-vue-mdc](https://github.com/tim15/brunch-vue-mdc) fails as well. So it might be a problem with vue-brunch / brunch / vueify
